Sources/Extensions/Alamofire/SessionManager+Extensions.swift

```diff
@@ -26,19 +26,73 @@ import RxAlamofire
 
 public extension Reactive where Base: SessionManager {
 
+    
+    /// Creates an observable of the `Request`.
+    ///
+    /// - Parameters:
+    ///   - method: Alamofire method object
+    ///   - url: An object adopting `URLConvertible`
+    ///   - parameters: An array of dictionaries containing all necessary options
+    ///   - encoding: The kind of encoding used to process parameters
+    ///   - headers: A dictionary containing all the additional headers
+    /// - Returns: An observable of the `Request`
+    public func request(_ method: Alamofire.HTTPMethod,
+                        _ url: URLConvertible,
+                        parameters: [Any]? = nil,
+                        encoding: ParameterEncoding = URLEncoding.default,
+                        headers: [String: String]? = nil
+        )
+        -> Observable<DataRequest>
+    {
+        return Observable.deferred {
+            var urlRequest = URLRequest(url: try url.asURL())
+            urlRequest.httpMethod = method.rawValue
+            urlRequest.allHTTPHeaderFields = headers
+            urlRequest.httpBody = try? JSONSerialization.data(withJSONObject: parameters, options: [])
+            
+            if urlRequest.value(forHTTPHeaderField: "Content-Type") == nil {
+                urlRequest.setValue("application/json", forHTTPHeaderField: "Content-Type")
```

Comment:
can you use this https://github.com/Alamofire/Alamofire/blob/3e488dcd5caf8d76f3da79c7b6d7f4d784ca3296/Source/HTTPHeaders.swift#L294 ?